### ISSUE TITLE
Feature Request: Custom Trials/Subscription Delay to include initial amount modification

### ISSUE THREAD

**Issue by andrewlimaza** [strangerstudios/paid-memberships-pro#512]
A lot of user's seem to be confused with the 'Subscription Delay Add on' and 'Custom Trial' that it only affects the recurring amount and not the initial amount.

Some user's would like to delay the membership for X days including the initial amount.

**Comment by andrewlimaza** [comment]
They would like a delay from the initial amount as a starting point and not the recurring amount.

**Comment by eighty20results** [comment]
This would be a violation of the terms of service for the payment gateways (You can only charge a card when you have the customer taking the action.). That is, unless they explicitly agree to a different behavior (would need multiple ways to verify identity of signor).  Then we'd have to store the payment method - legally, and within the requirements of the appropriate PCI-DSS standard (which is something few WordPress.org based sites have the ability to comply with), and push the initial payment on the agreed upon date. 

Basically, this isn't something I think we should touch (the liabilities are enormous!)

**Comment by andrewlimaza** [comment]
Thanks for this @eighty20results - wasn't aware of the legality. Maybe we can have a blog post regarding this to clear things up for user's asking. :)

**Comment by cdgdigital** [comment]
Although I don't link much this approach, but a possible "way around" for this issue could be to charge only $1 for the first month instead of free, so it can run trough payment gateways, and charge full on next payments, unless members cancel subscription.

**Comment by strangerstudios** [comment]
We've been discussing merging the subscription delay and set expiration date add ons into the core plugin by way of a new pricing wizard/setup for levels. Jessica did some work on this, and then we decided it needed more thought.

**Comment by andrewlimaza** [comment]
I think the workaround will be to only charge $1 for intial and this can be as seen as a card verification fee maybe?

**Comment by dparker1005** [comment]
It is now possible to modify the delay length of recurring payments, even when charging $0 for the initial payment. Alternatively, a "trial level" can be used to give a free trial, an initial payment amount, and a different recurring payment amount.
